IntroducingMethods/src/cst1201/IntroducingMethods.java

```diff
@@ -3,42 +3,65 @@
 import java.util.Scanner;
 
 /**
- * TODO: Description of your class.
  *
- * @author TODO: Your name here.
+ * @author Sameen Qaiser
  */
 public class IntroducingMethods {
 
+    /**
+     * @param args the command line arguments
+     */
     public static void main(String[] args) {
-
-        // TODO: Fill in the body with your code
+// TODO: Fill in the body with your code
+        Scanner keyboard = new Scanner(System.in);
+        System.out.print("Enter the string: ");
+        getInputString(keyboard);
     }
 
-    /**
-     * Given a Scanner, prompts the user for a String.  If the user enters an empty
-     * String, reports an error message and asks for a non-empty String.  Returns the
-     * String to the calling program.
-     * @param inScanner The Scanner to use to prompt the user.
-     * @return The resulting String from the user.
-     */
     private static String getInputString(Scanner inScanner) {
         // TODO: Fill in the body
+        String s = inScanner.nextLine();
+        if (s.length() > 0) {
+            getWordCount(s);
```

Comment:
This method should not be called from here. `getInputString()` only gets the input string. It doesn't also count words.